pkg/streamingpromql/operators/aggregations/aggregation.go

```diff
@@ -298,10 +298,12 @@ func (a *Aggregation) emitAnnotation(generator types.AnnotationGenerator) {
 }
 
 func (a *Aggregation) Prepare(ctx context.Context, params *types.PrepareParams) error {
+	// The wrapping operator is responsible for preparing the operators providing parameters, if any.
```

Comment:
I'm not sure I get the `preparing the operators providing parameters` part. Does it mean to prepare params it passing down?

Reply:
How's what I've added in e113f1c?